index.html

```diff
@@ -1,14 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="pages/index.css">
+    <meta charset="utf-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <link rel="stylesheet" href="pages/index.css" >
     <title>Pink App</title>
   </head>
   <body class="page">
     <div class="page__container">
-      <p>The repository is created with the purpose of learning adaptive web design and team collaboration using Git and GitHub.</p>
+      <p>
+        The repository is created with the purpose of learning adaptive web
+        design and team collaboration using Git and GitHub.
+      </p>
+      <section class="research">
+        <picture>
+          <source
+            srcset="images/research-timer-tablet.png"
+            media="(max-width: 660px)"
+          >
+          <img
+            src="images/research-timer-desktop.png"
+            alt="Картинка таймера"
+            class="research__image"
+          >
+        </picture>
+
+        <div class="research__texts">
+          <h3 class="research__title">Эффект на 24 часа!</h3>
+          <p class="research__paragrah">
+            Британские ученые провели исследования и доказали, что положительный
+            эффект от использования приложения длится целые сутки!
+          </p>
+          <a href="#" class="research__link">Узнать больше об исследовании</a>
```

Comment:
@Woolfs1423 Let's add the `target` attribute with the value `_blank` to this link, as it is likely that the link is intended to lead to an **external resource** with the full text of the application research.